content/about/index.md

```diff
@@ -1,9 +1,72 @@
 ---
-title: "About"
+title: "About Us"
 author: "Zap"
 date: 2018-05-25T18:07:23+05:30
-cover_image: "/images/image1.jpeg"
+cover_image: "/images/aboutus/zap.jpg"
 ---
 
-### Coming Soon....
+<div class="container">
+
+	<p>
+		<i>"Every word is not an abbreviation"</i>. Zap is a group of four members. We are studying in Northern India Engineering College and pursuing Btech degree. We all love to implement our theoretical knowledge into our real life. We also love to write blogs on what we learn. We have also represented India at Oulu HealthHack Hackathon at Finland. We are on 100Days of Code Challenge.
+	</p>
+
+	<h4 style="text-align:center;"> Team Members </h4>
```

Comment:
Move this `<h4>` outside `p` tag.
Move the `br` tag
And wrap this `h4` inside row column, like this

``` html
</p>
<div class="row">
		<div class="col" style="text-align:center; width: 100%" >
			<h4 > Team Members </h4>
		</div>
	</div>

	<div class="row">

	    <div class="col s4">
	 	  	<img class=" circle responsive-img " src="/images/aboutus/sarso.jpeg">
```